.changeset/four-bats-kick.md

```diff
@@ -0,0 +1,7 @@
+---
+"@getodk/scenario": patch
+"@getodk/xforms-engine": patch
+"@getodk/xpath": patch
```

Comment:
Nit: new functionality feels like minor? Not a big deal either way!

Reply:
Agree. I was partly basing this on precedent in 1ac6b10, which surprised me but seemed to be oriented around targeting some set of functionality for certain 0.x milestones. Not sure if that was the right inference, or if it's still pertinent.

Reply:
Interesting, I see! I think I was thinking of the API addition as internal but it’s not, really. So that one should have been minor for the engine and dependents?

This one feels unambiguous to me because it’s introducing user-facing functionality.